addons/common/functions/fnc_unloadPersonLocal.sqf

```diff
@@ -13,7 +13,7 @@
 //#define DEBUG_MODE_FULL
 #include "script_component.hpp"
 
-#define GROUP_SWITCH_ID QUOTE(FUNC(loadPerson))
+#define GROUP_SWITCH_ID QFUNC(loadPerson)
```

Comment:
Check out this file here: https://github.com/acemod/ACE3/blob/master/addons/main/script_debug.hpp